### PR TITLE
More resources given to slurmjobs

### PR DESCRIPTION
**Pull request by MatildaAslin**
I have noticed that Proj and Rep jobs fail quite often lately. This happens to HiSeqX runs with quite a lot of samples (~ 20 per lane).

### CODE REVIEW

File: aeacus-reports.pl [hunk 1]
```diff
@@ -174,9 +174,10 @@ =head1 DESCRIPTION
 					       EXECDIR=>$rfPath,      # Directory from which to run the script
 					       NAME=>"$proj-$rfShort",# Name of job, also used in script name
 					       PROJECT=>$uProj,       # project for resource allocation
-					       TIME=>"0-06:00:00",    # Maximum runtime, formatted as d-hh:mm:ss
+					       TIME=>"0-15:00:00",    # Maximum runtime, formatted as d-hh:mm:ss
 					       QOS=>$uQos,            # High priority
 					       PARTITION=>'core',      # core or node (or devel));
+                           CORES=>'2',
```
Comment (johandahlberg) [inline aeacus-reports.pl]: The indentation looks a bit weird here.

**Comment by johandahlberg** [comment]
Looks good - want to take a look at the indentation? I see it wasn't introduced here, but we might as well fix it when we're in this part of the code.

**Comment by MatildaAslin** [comment]
Sure! :+1:

**Comment by johandahlberg** [comment]
Great!

**Pull request merged**